src/main/groovy/edu/oregonstate/mist/osuevents/db/LocalistDAO.groovy

```diff
@@ -302,7 +302,7 @@ class LocalistDAO {
 
 @JsonIgnoreProperties(ignoreUnknown=true)
 class Filters {
-    @JsonProperty("event_counties")
+    @JsonProperty("event_county")
```

Comment:
The filter name is different for production and staging :(

Updating it to the production name which will break it for staging. I asked localist to update the staging name to match production.